omod/src/main/webapp/pages/metadata/roles/templates/purgeRoleDialog.gsp

```diff
@@ -0,0 +1,13 @@
+<div class="dialog-header">
+    <h3>${ui.message('adminui.role.purge.title')}</h3>
+</div>
+<div class="dialog-content">
+    <h4>
+        ${ui.message('uicommons.delete')} &quot;{{ role.name }}&quot;?
```

Comment:
Can you use a place holder in the message.properties file to insert the name? Otherwise this won't translate well in some languages where the ordering of the translated word differs, Try something like
my.message = Retire {0} and then you will need to replace the place holder({0}) with the role name in JS